Client/src/main/java/org/projectcardboard/client/models/gui/UIConstants.java

```diff
@@ -39,6 +42,9 @@
     
     static{
         try {
+            System.out.println(Screen.getPrimary().getBounds().getWidth());
```

Comment:
What are these three lines for? If we want to keep them, combine them into a single print statement with something like "Native screen resolution: width x height"

Reply:
thats just me being sloppy in removing all of my print statements :/